src/pages/auth/components/Spinner.jsx

```diff
@@ -0,0 +1,13 @@
+import { RotatingLines } from 'react-loader-spinner'
+
+export const Spinner = () => {
+  return (
+    <RotatingLines
+      strokeColor='white'
+      strokeWidth='5'
+      animationDuration='0.75'
+      width='20'
+      visible={true}
```

Comment:
visible={true}
visible 

the same, no need set ={true}

Reply:
done